cpp/unit/ability.h

```diff
@@ -42,17 +43,14 @@ using ability_set = std::bitset<16>;
 using ability_id_t = unsigned int;
```

Comment:
Usually you do it by adding a final member, `::MAX`, to the enum

enum class foo {
   val,
   bar=20,
   rofl,
   test,
   MAX
};

foo_size = foo::MAX